### DISCUSSION TITLE
Social Auth Logging Me in As Admin User

### DISCUSSION THREAD

**Discussion by pythoncreate** [python-social-auth/social-core#1327] [Q&A]
I'm having some really weird issues i'm trying to troubleshoot.

I've created a few dummy facebook and twitter accounts.

I can't figure out how, why or when this is happening.  BUT, even after cleaning all caches, passwords, etc, if i login using Twitter auth, it is logging me in as the Admin, even if the twitter account is associated with a different email address.

Anyone know why this might be happening?

**Comment by kosytedy** [comment]
Hello @pythoncreate please I'd like to know if you found a solution. I'm currently facing same issue. Thanks.
